app/views/slack/callback.html.erb

```diff
@@ -0,0 +1,140 @@
+<input type="hidden" id="authenticity_token" value="<%= form_authenticity_token %>"/>
```

Comment:
Seems like this `callback.html.erb` file really isn't being used anywhere. Let me know if I missed something.